src/com/example/Demo_Subj/Room.java

```diff
@@ -16,11 +16,15 @@
 
     private Bitmap bitmapRoom;
     private int roomID;
+    private int lowerRoomID;
+    private int upperRoomID;
+    private int leftRoomID;
+    private int rightRoomID;
     List<Integer> attachedRooms;
 
     private List<Item> itemList;
 
-    public Room (Bitmap bitmapRoomR, int ID, Context ctx) {
+    public Room (Bitmap bitmapRoomR, int ID, int leftRoom, int rightRoom, int upperRoom, int lowerRoom, Context ctx) {
```

Comment:
ist das nicht übertrieben ne extra klasse dafür zu schreiben? ich würde es wenn dann eher in getter und setter auslagern